RecoBTag/CTagging/interface/CharmTagger.h

```diff
@@ -0,0 +1,63 @@
+#ifndef RecoBTag_CTagging_CharmTagger_h
+#define RecoBTag_CTagging_CharmTagger_h
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "CommonTools/Utils/interface/TMVAEvaluator.h"
+#include "RecoBTau/JetTagComputer/interface/JetTagComputer.h"
+#include <mutex>
+#include "FWCore/Utilities/interface/ESInputTag.h"
+#include "RecoBTag/SecondaryVertex/interface/CombinedSVSoftLeptonComputer.h"
+#include "DataFormats/BTauReco/interface/TaggingVariable.h"
+#include "RecoBTau/JetTagComputer/interface/JetTagComputerRecord.h"
+
+//DEBUGGING
+#include "TFile.h"
+#include "TNtuple.h"
+#include <memory>
+#include <map>
+
+/** \class CharmTagger
+ *  \author M. Verzetti, U. Rochester, N.Y.
+ *  copied from ElectronTagger.h
+ */
+
+class CharmTagger : public JetTagComputer {
+public:
+  /// explicit ctor 
+	CharmTagger(const edm::ParameterSet & );
+	~CharmTagger();//{}
+  virtual float discriminator(const TagInfoHelper & tagInfo) const override;
+	virtual void initialize(const JetTagComputerRecord & record) override;
+	
+	typedef std::vector<edm::ParameterSet> vpset;
+	
+	struct MVAVar {
+		std::string name;
+		reco::btau::TaggingVariableName id;
+		size_t index;
+		bool has_index;
+		float default_value;
+	};
+
+private:
+  mutable std::mutex mutex_;
+	[[cms::thread_guard("mutex_")]] std::unique_ptr<TMVAEvaluator> mvaID_;
```

Comment:
We SHALL move to GBRForest for any new code (and migrate fast any old one!)
TMVAEvaluator shall not be allowed anymore in production code